superset/assets/visualizations/EventFlow.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import {
+  App,
+  withParentSize,
+  cleanEvents,
+  TS,
+  EVENT_NAME,
+  ENTITY_ID,
+} from '@data-ui/event-flow';
+
+/*
+ * This function takes the slice object and json payload as input and renders a
+ * responsive <EventFlow /> component using the json data.
+ */
+function renderEventFlow(slice, json) {
+  const container = document.querySelector(slice.selector);
+  const hasData = json.data && json.data.length > 0;
+
+  // the slice container overflows ~80px in explorer, so we have to correct for this
+  const isExplorer = (/explore/).test(window.location.pathname);
```

Comment:
not sure if I understand why, but I'm wondering if using `slice.width()` and `slice.height()` would address the 80px issue?